iOSClient/Settings/Settings/NCSettingsView.swift

```diff
@@ -73,49 +74,67 @@ struct NCSettingsView: View {
-                .sheet(isPresented: $showPasscode) {
-                    PasscodeView(isLockActive: $model.isLockActive)
-                }
-                /// Enable Touch ID
-                Toggle(NSLocalizedString("_enable_touch_face_id_", comment: ""), isOn: $model.enableTouchID)
-                    .tint(Color(NCBrandColor.shared.getElement(account: model.session.account)))
-                    .font(.system(size: 16))
-                    .onChange(of: model.enableTouchID) { _ in
-                        model.updateTouchIDSetting()
-                    }
-                /// Lock no screen
-                Toggle(NSLocalizedString("_lock_protection_no_screen_", comment: ""), isOn: $model.lockScreen)
-                    .tint(Color(NCBrandColor.shared.getElement(account: model.session.account)))
-                    .font(.system(size: 16))
-                    .onChange(of: model.lockScreen) { _ in
-                        model.updateLockScreenSetting()
+                .disabled(NCBrandOptions.shared.enforce_protection)
+            }, header: {
+                Text(NSLocalizedString("_privacy_", comment: ""))
+            }, footer: {
+                Text(NSLocalizedString("_lock_cannot_disable_mdm_", comment: ""))
+            })
+
+            if model.isLockActive {
+                Section(content: {
+                    Group {
+                        // Change passcode
+                        Button(action: {
+                            showChangePasscode.toggle()
+                        }, label: {
+                            VStack {
+                                Text(NSLocalizedString("_change_lock_passcode_", comment: ""))
+                                    .tint(Color(NCBrandColor.shared.textColor))
+                            }
+                            .font(.system(size: 16))
+                        })
+                        /// Enable Touch ID
+                        Toggle(NSLocalizedString("_enable_touch_face_id_", comment: ""), isOn: $model.enableTouchID)
+                            .onChange(of: model.enableTouchID) { _ in
+                                model.updateTouchIDSetting()
+                            }
+                        /// Lock no screen
+                        Toggle(NSLocalizedString("_lock_protection_no_screen_", comment: ""), isOn: $model.lockScreen)
+                            .onChange(of: model.lockScreen) { _ in
+                                model.updateLockScreenSetting()
+                            }
+
+                        /// Reset app wrong attempts
+                        Toggle(NSLocalizedString("_reset_wrong_passcode_", comment: ""), isOn: $model.resetWrongAttempts)
+                            .onChange(of: model.resetWrongAttempts) { _ in
+                                model.updateResetWrongAttemptsSetting()
+                            }
+                            .font(.system(size: 16))
```

Comment:
Why use such specific point definitions instead of one of [the text styles given by the system](https://developer.apple.com/documentation/swiftui/font/textstyle/body)? Also applies to other occurrences in this block of added lines. I assume it would improve font size consistency.